lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -166,7 +170,13 @@ def check_file_request(url) # rubocop:disable Metrics/CyclomaticComplexity
         scratch[:patient_ids_seen] = patient_ids_seen | [resource.id] if resource_type == 'Patient'
 
         unless resource_is_valid?(resource: resource, profile_url: profile_url)
-          assert false, "Resource at line \"#{line_count}\" does not conform to profile \"#{profile_url}\"."
+          if first_error.key?(:line_number)
+            @invalid_line_count += 1
+          else
+            @invalid_line_count = 1
+            first_error[:line_number] = line_count
+            first_error[:messages] = messages.select { |message| ['error', 'warning'].include? message[:type] }
```

Comment:
I don't think you should need to filter the messages here. `resource_is_valid?` filters them based on however the validator was configured to filter them.

Reply:
Changed to message.dup